data/remnant/remnant missions.txt

```diff
@@ -1062,6 +1062,141 @@ mission "Remnant: Learn Sign Follow Up"
 
 
 
+mission "Remnant: Follow-up on Terminus Researchers A-1"
+	name "Check up on Wormhole Researchers"
+	description "Return to Bounty to check up on a group of wormhole researchers you met some time ago."
+	minor
+	source
+		government "Remnant"
+	destination "Bounty"
+	to offer
+		has "Remnant: Learning Sign 2: done"
+		has "Terminus exploration: done"
+		not "Deep: Remnant 0: offered"
+		random < 40
+	on offer
+		conversation
+			`Looking around at the Remnant spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you first got your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants.`
+			branch secrets
+				has "deep: did not reveal remnant"
+			`	Thinking back, it suddenly occurs to you that you helped retrieve a science drone for some researchers on Bounty - researchers who were studying the wormhole in Terminus. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes. It would only be a matter of time until someone figures out how to use the key stones either by luck or by research, and then the Remnant's secrecy would vanish.`
```

Comment:
Everywhere else I've seen them referenced, they've been called **Keystones** .

Reply:
There are two separate but related outfits. The primary ones - the ones the Remnant have, who actually know what they are and how to use them, are called Quantum Key Stones. Other species who have similarly functioning stones don't know their true purpose, and thus the name has - over time - degenerated to Keystones. So when talking about the stones in general, they use the true name for these things.

Reply:
What about capitalization then?